dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -51,48 +51,53 @@
   // TODO(suztomo): Add option to specify entry point classes
 
   /**
-   * Returns Maven coordinates for a BOM if specified; otherwise null. Example value: {@code
+   * Returns the Maven coordinates for a BOM if specified; otherwise null. Example value: {@code
    * com.google.cloud:cloud-oss-bom:pom:1.0.0-SNAPSHOT}
    */
   @Nullable
-  abstract String getBomCoordinates();
+  abstract String getBom();
```

Comment:
It might be nice (in a later PR!) to have a class representing a Maven coordinate tuple so we're not passing strings around.